src/Sulu/Component/Content/Metadata/Loader/StructureXmlLoader.php

```diff
@@ -74,7 +74,7 @@ class StructureXmlLoader extends AbstractLoader
      */
     public function __construct(
         private CacheLifetimeResolverInterface $cacheLifetimeResolver,
-        private PropertiesXmlParser $propertiesXmlParser,
+        private DeprecatedPropertiesXmlParser $propertiesXmlParser,
```

Comment:
the whole service will later be removed after Form and Template loader are implemented.